src/routes/data/ipfs.ts

```diff
@@ -107,6 +115,73 @@ export const getIPFSDataRoute = async ({
                 return badGateway("Could not upload data to IPFS");
             }
 
+            // Feth the object's current Carrot-Template tag
+            let templateTag;
+            try {
+                const getTags = new GetObjectTaggingCommand({
+                    Bucket: s3Bucket,
+                    Key: cid,
+                });
+                const tags = await s3Client.send(getTags);
+                templateTag = tags.TagSet?.find(
+                    (tag) => tag.Key === "Carrot-Template",
```

Comment:
Should we define a type for the available tags?

Reply:
I don't think we really need that personally